### ISSUE TITLE
Incorrect Zone column content in Lighting Summary - Daylighting HTML table 

### ISSUE THREAD

**Issue by asparke2** [NREL/EnergyPlus#5935]
Issue overview
--------------
In the HTML file of models with daylighting controls, the Lighting Summary - Daylighting table has a column labeled `Zone`, but this column contains the name of the `Daylighting:Controls` object instead.  Rather than changing the heading, it would be preferable to change the contents to actually be the `Zone` names.

Here's an example from the HTML file:

![image](https://cloud.githubusercontent.com/assets/2009290/20144706/ac40aebc-a65a-11e6-9f85-ccd156889fbf.png)

And the corresponding IDF:

```
  Daylighting:Controls,
    Lobby_Records_Flr_1_DaylCtrl,  !- Name
    Lobby_Records_Flr_1,     !- Zone Name
    SplitFlux,               !- Daylighting Method
    ,                        !- Availability Schedule Name
    Stepped,                 !- Lighting Control Type
    0.3,                     !- Minimum Input Power Fraction for Continuous or ContinuousOff Dimming Control
    0.2,                     !- Minimum Light Output Fraction for Continuous or ContinuousOff Dimming Control
    3,                       !- Number of Stepped Control Steps
    1.0,                     !- Probability Lighting will be Reset When Needed in Manual Stepped Control
    Lobby_Records_Flr_1_DaylRefPt1,  !- Glare Calculation Daylighting Reference Point Name
    270.0,                   !- Glare Calculation Azimuth Angle of View Direction Clockwise from Zone y-Axis {deg}
    22.0,                    !- Maximum Allowable Discomfort Glare Index
    ,                        !- DElight Gridding Resolution {m2}
    Lobby_Records_Flr_1_DaylRefPt1,  !- Daylighting Reference Point 1 Name
    0.04,                    !- Fraction of Zone Controlled by Reference Point 1
    375.0,                   !- Illuminance Setpoint at Reference Point 1 {lux}
    Lobby_Records_Flr_1_DaylRefPt2,  !- Daylighting Reference Point 2 Name
    0.04,                    !- Fraction of Zone Controlled by Reference Point 2
    375.0;                   !- Illuminance Setpoint at Reference Point 2 {lux}

  Daylighting:ReferencePoint,
    Lobby_Records_Flr_1_DaylRefPt1,  !- Name
    Lobby_Records_Flr_1,     !- Zone Name
    1.397000,                !- X-Coordinate of Reference Point {m}
    13.335000,               !- Y-Coordinate of Reference Point {m}
    0.762000;                !- Z-Coordinate of Reference Point {m}
```

### Details
Some additional details for this issue (if relevant):
 - Windows 7 64bit
 - EnergyPlus 8.6

**Comment by JasonGlazer** [comment]
Thanks for catching this @asparke2

**Comment by mjwitte** [comment]
Closed via #5945
